sbin/start-master.sh

```diff
@@ -27,7 +27,7 @@ fi
 # Any changes need to be reflected there.
 CLASS="org.apache.spark.deploy.master.Master"
 
-if [[ "$@" = *--help ]] || [[ "$@" = *-h ]]; then
+if [[ "$*" = *--help ]] || [[ "$*" = *-h ]]; then
```

Comment:
SC2199 (error): Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).